react/props_and_component_structure/README.md

```diff
@@ -54,24 +51,22 @@ Here, we will build a simple application that uses props to display a list of sh
 ```js
 constructor() {
   super();
-  this.state = {
-    products: [
+  this.products = [
```

Comment:
A couple of thoughts here:
1. Let's default to functional components wherever we can
1. I would prefer to assign this array to a `products` variable that is outside the component rather than introduce this type of pattern. That would mean changes to the code here and below (as well as some explanations referring to constructors, `this.products`, etc.)

Reply:
Understood. We can make this functional. I was continuing to use the class component to really drive home that their parent components should (by and large) be class components and their child components should be functional.